domains/dylank.json

```
@@ -0,0 +1,20 @@
+{
+  "owner": {
+    "username": "DylanK46",
+    "email": "kainth.dylan@gmail.com"
+  },
+  "record": {
+    "A": [
+      "",
+      "90.217.214.245"
+    ],
+    "A": [
+      "www",
+      "90.217.214.245"
+    ],
+    "A": [
+      "90.217.214.245",
+      "90.217.214.245"
+    ]
```

If you have this single ip as the A record, you just need one.

```suggestion
    "A": [
      "90.217.214.245"
    ]
```